spring-petclinic-microservices/spring-petclinic-visits-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+#    refresh-interval: 1800000
+#    secret-keys: cosmosdburi,cosmosdbkey,cosmosdbsecondarykey, redisuri, redispassword
+#    token-acquire-timeout-seconds: 60
+
+spring:
+  cloud:
+    azure:
+      profile:
+        tenant-id: ${AZURE_KEYVAULT_TENANTID}
+      credential:
+        client-id: ${AZURE_KEYVAULT_CLIENTID}
+        client-secret: ${AZURE_KEYVAULT_CLIENTKEY}
+      keyvault:
+        secret:
+          property-source[0]:
+            refresh-interval: 1800000
+            secret-keys: cosmosdburi,cosmosdbkey,cosmosdbsecondarykey,redisuri,redispassword
+          property-source-enabled: true
```

Comment:
This can be improved by setting the property-source-enabled default to true in the auto-configuration lib.